modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/Theme.kt

```diff
@@ -101,6 +102,7 @@ data class PocketCastsTheme(
     val isLight: Boolean,
 )
 
+@SuppressLint("ConflictingOnColor")
```

Comment:
I have suppressed this error as I don't want to mess with the theme colors.

> /Users/philip/git_automattic/pocket-casts-android/modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/Theme.kt:115: Error: Conflicting 'on' color for a given background [ConflictingOnColor from androidx.compose.material]
>         onBackground = colors.secondaryIcon01,
>                        ~~~~~~~~~~~~~~~~~~~~~~
> /Users/philip/git_automattic/pocket-casts-android/modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/Theme.kt:116: Error: Conflicting 'on' color for a given background [ConflictingOnColor from androidx.compose.material]
>         onSurface = colors.primaryInteractive01,
>                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~
> 
>    Explanation for issues of type "ConflictingOnColor":
>    In the Material color system background colors have a corresponding 'on'  color which is used for the content color inside a component. For example, a button colored primary will have onPrimary text. Because of this, it is important that there is only one possible onColor for a given color value, otherwise there is no way to know which 'on' color should be used inside a component. To fix this either use the same 'on' color for identical background colors, or use a different background color for each 'on' color.